app/src/main/kotlin/moe/koiverse/archivetune/ui/screens/playlist/PlaylistSuggestionSection.kt

```
@@ -0,0 +1,196 @@
+                                                    if (success) {
+                                                        val playlistName = viewModel.playlist.value?.playlist?.name
+                                                        val message = if (playlistName != null) {
+                                                            context.getString(R.string.added_to_playlist, playlistName)
+                                                        } else {
+                                                            context.getString(R.string.add_to_playlist)
+                                                        }
+                                                        Toast.makeText(context, message, Toast.LENGTH_SHORT).show()
+                                                    } else {
+                                                        Toast.makeText(context, R.string.error, Toast.LENGTH_SHORT).show()
+                                                    }
+                                                } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)